modules/ROOT/pages/tokenization-logs.adoc

```diff
@@ -0,0 +1,44 @@
+== Tokenization Service Logs
+
+You can use the tokenization summary message that is returned in the Runtime Fabric logs to determine traffic counts. To retrieve the logs, follow the instructions in the xref:1.2@runtime-fabric::runtime-fabric-logs.adoc[Runtime Fabric documentation] for configuring log forwarding.
```

Comment:
no "the" before Runtime Fabric logs. I'd write the what before the how:
"To determine traffic counts (what are traffic counts?), use the tokenization summary message that is returned in Runtime Fabric logs.  To retrieve the logs, configure log forwarding in Runtime Fabric (Settings? or...?). then put the link in the See Also.